moshi/src/main/java/com/squareup/moshi/internal/Util.java

```diff
@@ -643,4 +662,12 @@ public static JsonDataException unexpectedNull(
     }
     return new JsonDataException(message);
   }
+
+  // Public due to inline access in MoshiKotlinTypesExtensions
+  public static <T> Class<T> wrap(Class<T> type) {
```

Comment:
Maybe `boxIfNeeded`? or `boxIfPrimitive`? Without looking at the implementation I had no idea what this was going to do.

Reply:
I was just using the name that guava's Primitives.java used, but can rename

Reply:
02848bd